openedx/core/djangoapps/oauth_dispatch/api.py

```diff
@@ -31,17 +30,15 @@ def create_dot_access_token(request, user, client, expires_in=None, scope=None):
             u'refresh_token': u'another string',
             u'token_type': u'Bearer',
             u'expires_in': 36000,
-            u'scope': u'default',
+            u'scope': u'profile email',
```

Comment:
Would it be better to use an Enum for our scopes than to use magic strings?

Reply:
This is listed as an example in the docstrings.
We need to iterate on how scopes are defined/managed/etc.  Right now, they are just listed in Django settings in common.py.